### PR TITLE
Add `11241036:modular-plaza-system`

### PR DESCRIPTION
**Pull request by sebamarynissen**
This PR adds [11241036's Modular Plaza System](https://community.simtropolis.com/files/file/31351-modular-plaza-system-mps/), as well as a bunch of its dependencies that weren't available in sc4pac yet. It provides variants for all of its 13 texture options.

A few things to note here:

 - It looks like 1 lot (MPS Square Column) is missing dependencies. It doesn't look like I have missed it, as I downloaded all of the [listed dependencies](https://community.simtropolis.com/files/file/31351-modular-plaza-system-mps/) separately and the props still didn't show up. I PM'ed 11241036 to see if he could verify.
 - It looks like the interactive sc4pac interface cannot handle 10 or more variant options as it doesn't allow you to select option 1 in this case - at least on Windows.
 - The `rdg:prop-pack-1` was already included in @Zasco's [sc4pac channel](https://github.com/Zasco/sc4pac-channel/blob/master/src/rdg/prop-pack-1.yaml) I chose to include it here to make it available on the default sc4pac channel as well, as that feels the natural thing to do with prop packs.

### CODE REVIEW

File: src/yaml/11241036/modular-plaza-system.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
+group: "11241036"
+name: modular-plaza-system
+version: "2"
+subfolder: 660-parks
+info:
+  summary: Modular Plaza System
+  description: |
+    This plugin contains a good number of modular lots that will allow you to create your custom plaza, sidewalk, or seawall. Fountains, ponds, benches, seawalls, market stalls, trees – nearly everything is available, Most of them are purely decorational (while having park and landmark effect), only the market stalls will yield a bit money. All lots have been designed in a way that they are rather simple, thereby being able to be combined with each other easily. All lots can be found in the park menu, just below the default Maxis parks.
+
+    Submenu support is baked in. To take advantage of it, you have to install `memo:submenus-dll`, after which the lots can be found in the Parks > Plazas submenu.
```
Comment (memo33) [inline src/yaml/11241036/modular-plaza-system.yaml:10]: You can use the synax ``` `pkg=memo:submenus-dll` ``` to create a link.

**Comment by memo33** [comment]
Oh my… More than 50 dependencies and still something's missing. I'm inclined to view it as the responsibility of the original author to list the complete dependencies on the STEX.

**Comment by sebamarynissen** [comment]
Yeah, let me tell you, this one was a true pita to add. 😅 Nothing compared to the mattb325 packs. I've contacted [11241036](https://community.simtropolis.com/profile/353048-11241036/) about the dependency issue, let's see what he has to say about it.

**Comment by memo33** [comment]
> Would it make sense to provide a global variant for the preferred resolution? What should we call it in this case?

I think it's fine to keep it as a per-package variant, so that one can choose case by case. (In fact, I don't understand why this is an option at all. The BATter should have just chosen the more appropriate resolution for each individual prop or model.)

**Comment by sebamarynissen** [comment]
I got a message back from 11241036 and turns out he indeed forgot to add a dependency. I'll update the package with it.

**Comment by sebamarynissen** [comment]
@memo33 ready for review

**Comment by memo33** [comment]
Thanks a lot. Any chance the MPS dependency list gets shorter with your updated scripts? Otherwise, I'm happy to merge this as is. (I've made small changes, mainly renaming a few packages a bit.)

**Comment by sebamarynissen** [comment]
I never used the tracking script for finding the dependencies as they were explicitly listed on the STEX. I did use the tracking script to find the *missing* dependencies though, but that has been fixed in the meantime. So no, unfortuntaly the dependency list won't get any shorter, but luckily we have sc4pac to handle those for us now.

**Comment by memo33** [comment]
Ah, I assumed you used the tracking script to resolve all the BSC Common Dependencies and Girafe Flora dependencies, as they are not listed on the STEX.

**Comment by sebamarynissen** [comment]
Ah, could be. It's so long ago. Let me quickly run the script, I'm guessing a few can be shaved off indeed.

**Comment by sebamarynissen** [comment]
Alright, I've removed the unneeded dependencies. I opened the city where I had plopped all the lots and no brown boxes appeared, indicating that the removal is ok.

**Comment by memo33** [comment]
Excellent. Thank you. Apologies for sitting on it for so long.

@Zasco With this PR merged, you could remove `rdg:prop-pack-1` from your channel.

**Pull request merged**